keps/prod-readiness/sig-storage/2324.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 1412
+stable:
```

Comment:
should be `alpha`.

you can assign me to this one, though it looks like you are targeting 1.22?

Reply:
Updated, and this has been assigned.

Correct. The hope is that we can have a working design this quarter, and alpha implementation will be targeting 1.22.